src/helpers/utilities.js

```diff
@@ -590,3 +590,13 @@ export const logSingleStore = (store, prev) => {
     }
   });
 };
+
+/**
+ * Check is two numbers are within the given tolerance of one another
+ * @param {a} number
+ * @param {b} number
+ * @param {tolerance} tolerance value
```

Comment:
you could also add a returns type here to convey that this function returns a boolean value.